recipes/elektronn/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "elektronn" %}
+{% set version = "1.0.13" %}
+{% set sha256 = "e5ba5dd4c5333576dde668977b9cced446eab5fdd3219d91d75550d1e8d5e2fc" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/e/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [not linux64 or not py27]
+
+requirements:
+  build:
+    - python <3
+    - setuptools
+    - cython
```

Comment:
You probably want to pin cython to `>=0.23` as is listed [in the setup.py for 1.0.13](https://github.com/ELEKTRONN/ELEKTRONN/blob/master/setup.py#L50)